jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -0,0 +1,68 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(name: 'VERSION', description: 'The version of Data Prepper', trim: true)
+        string(name: 'DATA_PREPPER_BUILD_NUMBER', description: 'The build number of the Data Prepper build from GitHub to release.', trim: true)
+    }
+    stages {
+        stage('Download Archives') {
+            steps {
+                script {
+                    archivePath = "${DATA_PREPPER_ARTIFACT_STAGING_SITE}/${VERSION}/${DATA_PREPPER_BUILD_NUMBER}/archive"
+
+                    sh 'mkdir archive'
```

Comment:
Let's use `dir` instead of `mkdir` https://github.com/opensearch-project/opensearch-build/blob/main/vars/createReleaseTag.groovy#L18